### PR TITLE
Simplify dependencies

### PR DESCRIPTION
**Pull request by bpedroza**

### CODE REVIEW

File: phpunit.xml.example [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false">
+    <testsuites>
+        <testsuite name="Application Test Suite">
```
Comment (hendeavors) [inline phpunit.xml.example]: Maybe name this Intergration Test Suite, I'm not sure if we really have any true unit tests in the repo.  Also can you add the verbose="true"

File: composer.json [hunk 2]
```diff
@@ -27,7 +26,7 @@
         }
     },
     "require-dev": {
-        "orchestra/testbench": "3.2.*"
+        "phpunit/phpunit": "^6.0"
```
Comment (hendeavors) [inline composer.json]: You should be ok to add an or for versions higher than 6. That way higher php versions can get an up to date phpunit install: `^6.0 || ^7.0`

**Review by hendeavors — CHANGES_REQUESTED** [review]

File: phpunit.xml.example (hunk 1 above)
Comment (bpedroza) [reply phpunit.xml.example]: Updated

File: composer.json (hunk 2 above)
Comment (bpedroza) [reply composer.json]: Updated.

**Review by hendeavors — APPROVED** [review]

**Pull request merged**